src/EditorFeatures/VisualBasicTest/EndConstructGeneration/EndConstructCommandHandlerTests.vb

```diff
@@ -112,6 +112,56 @@ End Module</code>.Value.Replace(vbLf, vbCrLf)
             VerifyAppliedAfterReturnUsingCommandHandler(code, {4, -1}, expected, {5, 8})
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.EndConstructGeneration)>
+        <WorkItem(2858, "https://github.com/dotnet/roslyn/issues/2858")>
+        Public Sub AddParenthesesToArgumentListOnReturn_1()
+            Dim code = <code>Public Class C
+    Public Sub M(arg As String)
+        M ""
+    End Sub
+End Class</code>.Value.Replace(vbLf, vbCrLf)
```

Comment:
There is/was .NormalizedValue as an extension for this. Or just use multi-line string literals now that they're a thing.